### ISSUE TITLE
Call for functionality (and testing)

### ISSUE THREAD

**Issue by thebigmunch** [thebigmunch/gmusicapi-wrapper#1]
While I plug away at a more extensive and varied logging/exception system, get the online documentation in place, and handle a bunch of small issues:

* What kind of functionality do people want to see in a convenience wrapper like this?
* Is there functionality in something you've written that could be ported into gmusicapi-wrapper?
* Is there anything in gmusicapi-wrapper that could be changed to help something you're doing?

**Comment by thebigmunch** [comment]
I'll start it off with this: a unified client interface. It could be incredibly useful to combine functionality from different Google Music/gmusicapi client interfaces as well as simply be easier/more convenient.

**Comment by ScottESanDiego** [comment]
I think you want to make the "path = path.decode('utf8')" statement in gmusicapi_wrapper.py unconditional.  I don't see why you'd NOT want to utf8 decode it, and with it in the conditional as it is currently, I get errors on uploading some files (UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 3").

**Comment by thebigmunch** [comment]
The conditional is brokeb because I missed changing str back to unicode after I was done playing around with gmusicapi and Python 3, since I was thinking about releasing this as compatible with Python 3 from the start. But I changed my mind on that since Python 3 support in gmusicapi is still a long way away.

I've actually got this fixed up locally, and I should actually be working/releasing my gmusicapi stuff this week. Sorry for the issue, and thanks for the comment.
